DiscordBotApis/Data/GuildManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBotApis.Models;
+
+namespace DiscordBotApis.Data
+{
+    public class GuildManager : IGuildRepository
+    {
+        private GuildContext _guildContext;
+
+        public GuildManager(GuildContext context)
+        {
+            _guildContext = context;
+        }
+
+        public Guild GetGuildById(ulong id)
+
+        {
+            Console.WriteLine(_guildContext.Guilds);
```

Comment:
Guessing this line was for debugging?  Can we please remove before merge?

Reply:
Yes, good call... fixed